app/js/lib/foxbox.js

```diff
@@ -327,6 +326,25 @@ export default class Foxbox extends Service {
     });
   }
 
+  performServiceCommand(id, command, method = 'GET', state = undefined) {
```

Comment:
__question:__ What is your plan to perform various service-specific commands? For now I've exposed generic ```performServiceCommand```. What do you think?